kibana/install.sh

```diff
@@ -0,0 +1,24 @@
+mkdir -p -m 755 ${KIBANA_HOME}/installedPlugins/origin-kibana
+cd ${KIBANA_HOME}/installedPlugins/origin-kibana
+wget -q ${AOP_KIBANA_PLUGIN_REPO}/releases/download/${AOP_KIBANA_PLUGIN_VER}/origin-kibana-${AOP_KIBANA_PLUGIN_VER}.tgz
+tar -xvzf origin-kibana-${AOP_KIBANA_PLUGIN_VER}.tgz
+rm origin-kibana-${AOP_KIBANA_PLUGIN_VER}.tgz
+
+# Kibana starts up slowly because it tries to optimize and cache bundles
+# so we start it up as part of install and then stop it
+mv ${KIBANA_HOME}/config/kibana.yml ${KIBANA_HOME}/config/hidden_kibana.yml
+touch ${KIBANA_HOME}/config/kibana.yml
+
+${KIBANA_HOME}/bin/kibana > ${KIBANA_HOME}/kibana.out &
+pid=$!
+
+until [ -n "$(grep 'Optimization of bundles for kibana and statusPage complete' ${KIBANA_HOME}/kibana.out)" ]; do
```

Comment:
It shouldn't wait forever...., the other side of this is if we don't do it here Kibana would do it upon normal start up and then it if there is a chance this would happen it would wait forever when trying to start up for a customer.